pbcopy

```diff
@@ -0,0 +1,435 @@
+{% extends "base.html" %}
```

Comment:
this file not used?

Reply:
i added made this file but deleted it so its not in the current branch

Reply:
i accidentally added this file in a previous commit then removed it in a later commit

Reply:
its still there